src/main/java/io/iron/ironworker/client/builders/TaskOptionsObject.java

```diff
@@ -39,6 +39,12 @@ public TaskOptionsObject delay(int delay) {
 
         return this;
     }
+    
+    public TaskOptionsObject encryptionKeyFile(String encryptionKeyFile){
+    	options.put("encryptionKeyFile", encryptionKeyFile);
```

Comment:
There are also other properties such "delay", "label", "cluster", etc. and they are without corresponding constants. I made similarly.